sources/packages/backend/apps/api/src/route-controllers/student-appeal/models/student-appeal.dto.ts

```diff
@@ -4,18 +4,32 @@ import {
   IsDefined,
   IsEnum,
   IsNotEmpty,
+  IsNumber,
   IsPositive,
   ValidateNested,
 } from "class-validator";
 import { StudentAppealStatus } from "@sims/sims-db";
+
 /**
  * DTO for student appeal request.
  */
 export class StudentAppealRequestAPIInDTO {
   @IsNotEmpty()
   formName: string;
   @IsDefined()
-  formData: unknown;
+  formData: StudentIncomeAppealRequest | PartnerIncomeAppealRequest;
```

Comment:
Same here on the context of dynamic behavior. 
Also I believe there are more appeal requests to come, so IMO we must just leave it the way it was. 

Correct me if wrong @andrewsignori-aot @guru-aot

Reply:
Yes you are right, currently we know these 2.

Reply:
In terms of dynamic data, as a general rule, the API must define types for whatever the API need to interact with. So far for these appeals (and hopefully all others to come), the API's responsibility is just to receive and save them (also validate with the `dryRun`) as they are to later also provide them as they were received to the workflow.
Short answer, please leave it as it was following what @dheepak-aot mentioned.